<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagesPane</class>
 <widget class="QMainWindow" name="MessagesPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window Messages</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="MessageWidget" name="messageWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actnLocate"/>
   <addaction name="separator"/>
   <addaction name="actnSave"/>
   <addaction name="separator"/>
   <addaction name="actnCapture"/>
   <addaction name="actnClear"/>
   <addaction name="separator"/>
   <addaction name="actnAutoExpand"/>
   <addaction name="actnFilter"/>
   <addaction name="actnHighlight"/>
  </widget>
  <action name="actnSave">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Messages</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actnAutoExpand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/auto_expand.png</normaloff>:/img/auto_expand.png</iconset>
   </property>
   <property name="text">
    <string>Auto Expand</string>
   </property>
   <property name="toolTip">
    <string>Automatically expand each new item</string>
   </property>
  </action>
  <action name="actnLocate">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/locate.png</normaloff>:/img/locate.png</iconset>
   </property>
   <property name="text">
    <string>Locate</string>
   </property>
   <property name="toolTip">
    <string>Shows the window in the hierarchy tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actnFilter">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/filter.png</normaloff>:/img/filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Include/exclude messages in the list</string>
   </property>
  </action>
  <action name="actnHighlight">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/highlight.png</normaloff>:/img/highlight.png</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
   <property name="toolTip">
    <string>Highlight messages in the list</string>
   </property>
  </action>
  <action name="actnCapture">
   <property name="text">
    <string>Start/Stop</string>
   </property>
   <property name="toolTip">
    <string>Starts/Stops capturing messages</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actnClear">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/list_clear.png</normaloff>:/img/list_clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clears the message list</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QTreeWidget</extends>
   <header>ui/custom_widgets/MessageWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
